examples/kubernetes/glusterd-ds.yaml

```diff
@@ -0,0 +1,86 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: glusterfs
+  labels:
+    glusterfs: daemonset
+  annotations:
+    description: GlusterFS DaemonSet
+    tags: glusterfs
+spec:
+  template:
+    metadata:
+      name: glusterfs
+      labels:
+        glusterfs: pod
+        glusterfs-node: pod
+    spec:
+      hostNetwork: true
+      containers:
+      - image: docker.io/gluster/glusterd2-nightly
+        imagePullPolicy: IfNotPresent
+        name: glusterfs
+        resources:
+          requests:
+            memory: 100Mi
+            cpu: 100m
+        volumeMounts:
+        - name: glusterfs-run
+          mountPath: "/run"
+        - name: glusterfs-lvm
+          mountPath: "/run/lvm"
+        - mountPath: /usr/local/var/lib/glusterd2
+          name: glusterd2-config
+        - name: glusterfs-etc
+          mountPath: "/etc/glusterfs"
+        - name: glusterfs-logs
+          mountPath: "/var/log/glusterfs"
+        - name: glusterfs-config
+          mountPath: "/var/lib/glusterd"
+        - name: glusterfs-dev
+          mountPath: "/dev"
+        - name: glusterfs-misc
+          mountPath: "/var/lib/misc/glusterfsd"
```

Comment:
`/var/lib/misc/glusterfsd` should probably `/var/lib/glusterfsd` ? Or maybe `/var/lib/gluster/...something`?

Reply:
This was done for geo replication file persistence and the path was `../misc`. Unless there is a change in the path keep it as it is